.markdownlint.yaml

```diff
@@ -3,3 +3,6 @@ MD013: false
 # mkdocs uses 4 spaces indent
 MD007:
   indent: 4
+MD024:
+  siblings_only: true
+MD034: false
```

Comment:
Would this break the links to the existing headers, e.g. https://github.com/argoproj/argo-workflows/blob/master/CHANGELOG.md#v350-rc1-2023-08-15?

Reply:
I'm not sure about the concerns, but the anchor to the header seems to work well: https://github.com/argoproj/argo-workflows/blob/428ddb118e826d322527a7a5c59e80418401ae1f/CHANGELOG.md#v350-rc1-2023-08-15

[`MD034`](https://github.com/DavidAnson/markdownlint/blob/main/doc/md034.md) is a lint rule about using url in markdown file. (ex. `https://example.com` in `*.md` will produce warning)

Reply:
I think it would be better to configure these inline for the whole file rather than globally.

As in, using:
```html
<!-- markdownlint-disable MD034 -->
<!-- markdownlint-configure-file { "MD024": { "siblings_only": true } } -->
```

Also some comments would be nice as to why these were disabled, as is convention